packages/ui-drawer-layout/src/DrawerLayout/DrawerContent/__tests__/DrawerContent.test.tsx

```diff
@@ -38,24 +39,21 @@ describe('<DrawerContent />', async () => {
     expect(drawerContent).to.exist()
   })
 
-  it('should not have a transition class if `shouldTransition` is set to false', async () => {
+  it('should not transition on mount, just on update', async () => {
```

Comment:
I refactored this test because it wasn't changed when we changed it's logic (twice since then actually).